docs/content/dagster-plus/references/limits.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Dagster+ Rate Limits | Dagster Docs"
+description: "Dagster+ Rate Limiting"
+---
+
+# Dagster+ Rate Limits
+
+Dagster+ enforces several rate limits to smoothly distribute the load. Deployments are limited to:
+
+- 40,000 user log events (e.g, `context.log.info`) per minute. This limit only applies to custom logs; system events like the ones that drive orchestration or materialize assets are not subject to this limit.
+- 35MB of events per minute. This limit applies to both custom events and system events.
+
+Rate-limited requests return a "429 - Too Many Requests" response. Dagster+ agents automatically retry these requests.
```

Comment:
maybe "agents and runs" or "agents and run workers"? many events are logged directly from the run worker but not sure if that distinction is relevant here.

Reply:
Yea that seems too nuanced for an average user.